scripts/get-members.ts

```diff
@@ -0,0 +1,33 @@
+import { db } from './firebase';
+
+let getNetId = (email: string): string => {
+  let pos = email.search('@');
+  return email.slice(0, pos);
+};
+
+const fs = require('fs');
+const dirPath: string = 'scripts/members/';
+fs.mkdirSync(dirPath, { recursive: true });
```

Comment:
How does fs.mkdirSync work?

Reply:
It just makes a directory with path `dirPath`. I think the `{ recursive: true }` argument just makes it create the directory recursively.